### DISCUSSION TITLE
已下载视频加入搜索框

### DISCUSSION THREAD

**Discussion by lyppiao** [ywmoyue/biliuwp-lite#365] [feature]
建议在“下载完成”界面，加个搜索框。用来搜索已下载的视频，这样比较方便。
如图：
![屏幕截图 2023-10-28 030205](https://github.com/ywmoyue/biliuwp-lite/assets/143381863/ff6a2003-774d-4cde-8616-e522d3dab210)

**Comment by lyppiao** [comment]
希望在导出视频时，可以自动把视频名字设置为文件名字，而不是现在的“导出的视频”。
如果有技术难度的话，可以把下载里的视频名，改为允许复制，这样也方便点。

**Reply by ywmoyue** [reply]
这个需求我转issue了，当前discussion我先关了

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/issues/539
